functions/pubsub/publish-message/pom.xml

```diff
@@ -100,6 +86,24 @@
       <version>${powermock.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <version>5.6.2</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
You really don't wish to mix JUnit4 & Junit5

Reply:
Done.